javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,167 @@
+// Flight booking full name function =============================================================================
+
+const getFullName = (firstName, lastName) => `${firstName} ${lastName}`;
+
+const fullName1 = getFullName('Rostyslav', 'Zhyvitskyi');
+const fullName2 = getFullName('Benjamin', 'Hughes');
+
+console.log(fullName1, fullName2);
+
+// Formal full name
+
+const getFullNameFormal = (firstName, lastName, sex, useFormalName) => {
+  if (firstName === '' || lastName === '') {
+    throw Error('Please make sure you have entered your first and last name');
+  }
+
+  if (useFormalName) {
+    return sex === 'male'
```

Comment:
What should happen if you have useFormalName but no sex? `getFullNameFormal('name', 'surname', true)`

Reply:
Fixed😊